### PR TITLE
feat: beta version of cli created

### PR DESCRIPTION
**Pull request by victordantasdev**
I created a CLI for tweeting from the terminal that can be installed globally via npm (as shown in the video) or can be run via npx.

P.S.: The package is ready, but it needs to be published on npm. This "ccsseraphini-cli" was a temporary name that I gave and posted on my personal account, but soon I will delete the package!

P.P.S.: The CLI description and README need tweaking, but obviously I'm not creative for that :))))

Issue #19 tasks:
- [x] be able to tweet from terminal
- [x] npx ccsseraphini "How can I learn Relay?"

https://user-images.githubusercontent.com/64330605/149665623-95f89a3f-21ef-4a55-979f-cb64165fbd5a.mp4

### CODE REVIEW

**Review by biantris — APPROVED** [review]
Good!

File: cli/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (noghartt) [inline cli/.gitignore:1]: We could insert `node_modules/` into the `.gitignore`  in the root folder, so isn't necessary to repeat it here.

File: cli/bin/cli.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  if (message === undefined || message === '-h' || message === '--help') {
+    showHelp();
+  } else if (message === '-v' || message === '--version') {
+    showVersion();
+  } else if (
+    (message.charAt(0) === '-' && message !== '-h') ||
+    (message.charAt(0) === '-' && message !== '--help') ||
+    (message.charAt(0) === '-' && message !== '-v') ||
+    (message.charAt(0) === '-' && message !== '--version')
+  ) {
```
Comment (noghartt) [inline cli/bin/cli.js:18]: What do you think to turn this conditional into:

```js
const validFlags = ['-v', '--version', '-h', ' --help'];

// ...
} else if (message.charAt(0) === '-' && !message.includes(validFlags)) {
  // ...
}
````

File: cli/index.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  if (color === 'yellow') {
+    return `${'\x1b[33m' + str + reset}`;
+  }
+
+  if (color === 'green') {
+    return `${'\x1b[32m' + str + reset}`;
+  }
+
+  return `${str + reset}`;
+}
```
Comment (noghartt) [inline cli/index.js:19]: I think that we can turn this color codes into an object, something like it:

```js
const reset = '\x1b[0m';
const colors = {
  red: '\x1b[31m',
  yellow: '\x1b[33m',
  green: '\x1b[32m',
};

// So, we can do something like it:
const resetedStr = `${str + reset}`;
return (colors[color] || '') + resetStr;
```

**Review by noghartt — APPROVED** [review]

**Comment by noghartt** [comment]
I think that we could turn this into a monorepo, creating a package to the `frontend` and other to `cli`, what do you think about it?

Also, can we move this repo to [ccseraphini](https://github.com/ccsseraphini)'s org?
cc @sibelius

**Comment by sibelius** [comment]
we can improve more on main branch

tks

**Pull request merged**